integrationTests/vm/testInitializer.go

```diff
@@ -89,7 +89,10 @@ func CreateEmptyAddress() []byte {
 
 // CreateMemUnit -
 func CreateMemUnit() storage.Storer {
-	cache, _ := storageUnit.NewCache(storageUnit.LRUCache, 10, 1)
+	size := uint32(10)
```

Comment:
might think to rename the `size` to `capacity` or something similar as now we have `size` and `sizeInBytes`. Apply this everywhere if you consider this suggestion relevant

Reply:
done, just about everywhere. Capacity replaced Size